UITests/UITests.App/MainPage.xaml

```diff
@@ -9,11 +9,17 @@
     Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
 
     <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="*" />
+            <RowDefinition Height="Auto" />
+        </Grid.RowDefinitions>
         <testhelpers:TestAutomationHelpersPanel />
 
-        <StackPanel HorizontalAlignment="Center" VerticalAlignment="Center">
-            <Button Content="Click Me" Click="Button_Click"/>
-            <TextBlock x:Name="textBlock"/>
+        <Frame x:Name="navigationFrame" />
```

Comment:
As called out in the main PR, thanks for adding a Frame here and starting to split this up! We definitely want to make it easier for folks to isolate and add new tests. 🎉🎉